### PR TITLE
feat(plugin): add plugin install command

### PR DESCRIPTION
**Pull request by zewa666**
this adds the basic command to install plugins upon asking the user which one to install.

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
@ahmedshuhel Please review. Thanks!

File: src/lib/installer/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 var ghdownload = require('download-github-repo')
```
Comment (ahmedshuhel) [inline src/lib/installer/index.js]: Could we change these `require` to es6 `import` statements and the module.exports to `export`? I know we have other lib modules that have still these `require`s. We may take an approach of converting an old module as we need a change there.

**Comment by ahmedshuhel** [comment]
As always @zewa666 's code is clean and looks good. I will give the command a try.

File: src/lib/installer/index.js (hunk 1 above)
Comment (zewa666) [inline src/lib/installer/index.js]: Sure no Problem, not at home but will change that tomorrow

**Comment by zewa666** [comment]
@ahmedshuhel I'm sorry it took me so long but now the changes have been supplied and I think it's ready for review

**Pull request merged**